website/static/js/comment.js

```diff
@@ -11,16 +11,27 @@ var koHelpers = require('./koHelpers');
 require('knockout.punches');
 require('jquery-autosize');
 ko.punches.enableAll();
-var Raven = require('raven-js');
 
 var osfHelpers = require('js/osfHelpers');
 var CommentPane = require('js/commentpane');
 var markdown = require('js/markdown');
+var waterbutler = require('./waterbutler');
 
 var nodeApiUrl = window.contextVars.node.urls.api;
+var nodeId = window.contextVars.node.id;
```

Comment:
Reusable modules should not depend on `contextVars`. Can you instead pass the `nodeId` as a parameter to the constructors that need it?